app/src/main/java/com/x8bit/bitwarden/ui/platform/components/button/BitwardenFilledIconButton.kt

```diff
@@ -40,21 +42,19 @@ fun BitwardenIconButtonWithResource(
         enabled = isEnabled,
     ) {
         Icon(
-            painter = iconRes.iconPainter,
-            contentDescription = iconRes.contentDescription,
+            painter = rememberVectorPainter(id = iconRes),
```

Comment:
How do we know this is a vector resource?

Reply:
I guess we technically don't know but all our icons should be vectors, right?